icicle/backend/cpu/src/field/cpu_vec_ops.cpp

```diff
@@ -803,12 +803,18 @@ REGISTER_SLICE_EXT_FIELD_BACKEND("CPU", cpu_slice<extension_t>);
 
 /*********************************** Highest non-zero idx ***********************************/
 template <typename T>
-eIcicleError cpu_highest_non_zero_idx(
-  const Device& device, const T* input, uint64_t size, const VecOpsConfig& config, int64_t* out_idx /*OUT*/)
+eIcicleError cpu_highest_non_zero_index(
```

Comment:
I would rename this to "internal" or maybe prefix with underscore rather than replace idx with index since it's confusing for readers who are not aware of it.